pkg/addon/templateagent/values.go

```diff
@@ -36,6 +36,15 @@ func ToAddOnRegistriesPrivateValues(config addonapiv1alpha1.AddOnDeploymentConfi
 	}, nil
 }
 
+func ToAddOnInstallNamespacePrivateValues(config addonapiv1alpha1.AddOnDeploymentConfig) (addonfactory.Values, error) {
+	if len(config.Spec.AgentInstallNamespace) == 0 {
+		return nil, nil
+	}
+	return addonfactory.Values{
+		"INSTALL_NAMESPACE": config.Spec.AgentInstallNamespace,
```

Comment:
Not sure if the install namespace should be a private value, if it is private, it should start with `__`, like`__INSTALL_NAMESPACE`

Reply:
this is done